src/components/uri/Visualization.tsx

```diff
@@ -514,34 +515,44 @@ function buildAfmResultSpec(
     visObj: IVisualizationObjectContent,
     visType: VisType,
     locale: Localization.ILocale,
-    canSortStackTotalValue?: boolean,
+    canSortStackTotalValue: boolean = false,
+    enableSortingByTotalGroup: boolean = false,
```

Comment:
we should put these to `featureFlags` object, otherwise `buildAfmResultSpec` signature would expand significantly
```
canSortStackTotalValue: boolean = 
enableSortingByTotalGroup: boolean = false,
```

Reply:
`canSortStackTotalValue` is not a FF.
I put both of them into a variable named `sortingConfigs`